utils/recipients.py

```diff
@@ -220,8 +227,25 @@ def validate_email_address(email_address):
         raise InvalidEmailError('Not a valid email address')
 
 
+def format_email_address(email_address):
+    return email_address.strip().lower()
+
+
 def validate_recipient(recipient, template_type):
     return {
         'email': validate_email_address,
         'sms': validate_phone_number
     }[template_type](recipient)
+
+
+def format_recipient(recipient):
+    try:
+        return validate_and_format_phone_number(recipient)
+    except InvalidPhoneError:
```

Comment:
Thinking about it more, I agree that it wasn’t that clear. Rewritten here: https://github.com/alphagov/notifications-utils/pull/16/files#diff-e3ab47367cb27430e6a12689b0b0a0f5R247